### PR TITLE
Alternative GLO

### PR DESCRIPTION
**Pull request by jakewesterberg**
Alternative glo task design with glo sequences split before and after controls. GLO trial counts are currently biased with more in the earlier (66%) than in the later block (33%). This is changed by altering the time for glo_duration_1 and glo_duration_2. As coded, task will occur habituation -> glo_1 -> randomized_control -> sequenced_control -> glo_2 (followed by RF mapping and optotagging). 

I also modified the trial counts in the randomized control to bias 4 of the 16 sequences instead of just 3.

### CODE REVIEW

File: pilot-scripts-allen/both-cohorts-day-11-ephysrecording-120min-drifting.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def generate_sequence(window, session_params, in_session_time, stimulus_counter,
             patterns_sequence           = []
             sequence['orientations']    = []
 
-            patterns_prob           = [ 3.0,  3.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  3.0,  1.0,  1.0,  1.0,]
+            patterns_prob           = [ 3.0,  3.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  3.0,  1.0,  1.0,  3.0,]
```
Comment (jeromelecoq) [inline pilot-scripts-allen/both-cohorts-day-11-ephysrecording-120min-drifting.py:326]: Is this change here on the non-alternative protocol intended?

Comment (jeromelecoq) [reply pilot-scripts-allen/both-cohorts-day-11-ephysrecording-120min-drifting.py:326]: Can you make you other files (pre-recordings) does not need to modify on this as well?

**Comment by jeromelecoq** [comment]
I am going to merge for now. But I suspect this could use some iteration.

File: pilot-scripts-allen/both-cohorts-day-11-ephysrecording-120min-drifting.py (hunk 1 above)
Comment (jakewesterberg) [reply pilot-scripts-allen/both-cohorts-day-11-ephysrecording-120min-drifting.py:326]: (1) Yes, this change is intended. There is a specific sequence in the randomized control that we would like more trials of regardless of whether we go down the current or alternative route.

(2) The other pre-recording files should not need to be changed as the subjects don't experience the control sequences during training. I will double-check to make sure nothing else needs to be changed in the training files.

**Pull request merged**